client/driver/docker.go

```diff
@@ -23,6 +24,34 @@ type DockerDriver struct {
 	fingerprint.StaticFingerprinter
 }
 
+type DockerDriverConfig struct {
+	ImageName     string           `mapstructure:"image"`
```

Comment:
Can we put comments on what the fields do. Also can the auth related things be in their own struct?